packages/server/lib/cloud/api/get_app_studio.ts

```diff
@@ -0,0 +1,137 @@
+import path from 'path'
+import os from 'os'
+import { ensureDir, copy, readFile } from 'fs-extra'
+import cloudApi from '.'
+import { StudioManager } from '../studio'
+import tar from 'tar'
+import { verifySignatureFromFile } from '../encryption'
+import crypto from 'crypto'
+import fs from 'fs'
+import fetch from 'cross-fetch'
```

Comment:
Doesn't need to be done in this PR but since we know this is executed in the node context, we could probably use `fetch` or `node-fetch` (which is what `cross-fetch` delivers under the hood) directly. This may also resolve some of the typing errors below.